### ISSUE TITLE
Psycho Pinball (EU/PAL) - heavily distorted music

### ISSUE THREAD

**Issue by guzzard** [MiSTer-devel/Genesis_MiSTer#151]
Hello! Thanks a lot for all the hard work you guys put into MiSTer! Awesome project!

I found that some music in Psycho Pinball is heavily distorted. This happens when running the game in EU/PAL mode. When forcing US/NTSC mode the music sound normal. This is a [PAL game](https://segaretro.org/Psycho_Pinball) that was never released in US etc, so I think EU/PAL mode should work?

Example video of problem: https://sendvid.com/93jmzhw8
In the example the core is the latest version with all settings reverted to default.

**Comment by birdybro** [comment]
Just tested this out. At first I thought it was because the region code used in the ROM header was "JUE" meaning universal. Then I set the Genesis core to use header detection and set priority with EU first in the chain, then saved settings, and rebooted the MiSTer to launch the Genesis core "clean".

What happens is interesting, the first load of the rom has the same exact problem. Upon resetting the game in the OSD, it works fine. Something about the initial load and play of the ROM causes this error to occur, regardless of region. Video demonstrating it below (VOLUME WARNING! LOUD!!!):

https://sendvid.com/91l7spqg

So the initial state of the core, for this particular ROM, is the cause of the problem (likely due to bad programming of the game itself). Maybe an initial RAM state for the core differs? I don't know enough to solve this, but decided to just test it out and see.

**Comment by guzzard** [comment]
@birdybro thanks for the info about the reset workaround :+1: :+1: . That will do fine until it's fixed permanently.

**Comment by guzzard** [comment]
@birdybro I tested using DDR3 as rom storage and it hides/solves this issue as well as you suggested it might in #157.

**Comment by birdybro** [comment]
Yup, there's likely something inaccurate with the initial RAM state on games booted when the OSD's ROM storage is set to SDRAM.
